src/main/java/edu/kis/vh/nursery/HanoiRhymer.java

```diff
@@ -2,16 +2,17 @@
 
 public class HanoiRhymer extends DefaultCountingOutRhymer {
 
-    int totalRejected = 0;
+    private int totalRejected = 0;
 
     public int reportRejected() {
         return totalRejected;
     }
 
```

Comment:
Dopisz tu @Override i bedzie git

Reply:
![image](https://user-images.githubusercontent.com/37260994/233378895-0bc5040d-35e9-4307-823b-1e74eb9ff28e.png)
kurde nie moge tego zrobic bo sety na branchu sa jakos dziwnie ustawione :/ moglbys to zrobic za mnie?

Reply:
Sory ale ja też mam inne repo do edycji ;/ i nie wiem jak to jest oceniane
Ty na dobrego brancha robisz tu pusha? Nie jestem pewny ale wygląda to tak jakbyś robił pusha na mastera a nie na branch "constants"